src/openrct2/actions/WallRemoveAction.hpp

```diff
@@ -53,7 +47,7 @@ struct WallRemoveAction : public GameActionBase<GAME_COMMAND_REMOVE_WALL, GameAc
     {
         GameAction::Serialise(stream);
 
-        stream << _x << _y << _baseHeight << _direction;
+        stream << _location.x << _location.y << _location.z << _location.direction;
```

Comment:
We should probably define the `<<` and `>>` operators on `TileCoordsXYZD`. Not necessarily in this PR, though.